integration-tests/application-server-integration-tests/src/test/java/co/elastic/apm/servlet/TomcatIT.java

```diff
@@ -55,7 +55,12 @@ public TomcatIT(final String tomcatVersion) {
             {"8.5-jre8-slim"},
             {"9-jre9-slim"},
             {"9-jre10-slim"},
-            {"9-jre11-slim"}
+            {"9-jre11-slim"},
+            {"9.0.39-jdk14-openjdk-oracle"}
+// Note: not compatible yet
```

Comment:
What is not compatible?

Reply:
Test fails on `testAllScenarios`, I think this relate to tomcat inner changes(support jakarta ee instead of java ee). In logs I didn't find applying of `ServletApiAdvice` instrumentation. 
```
2020-11-29 19:36:53,547 [main] WARN  com.networknt.schema.JsonMetaSchema - Unknown keyword regexProperties - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2020-11-29 19:36:53,557 [main] WARN  com.networknt.schema.JsonMetaSchema - Unknown keyword deprecated - you should define your own Meta Schema. If the keyword is irrelevant for validation, just use a NonValidationKeyword
2020-11-29 19:36:53,991 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Check file /home/admin/Documents/Projects/apm/apm-agent-java/integration-tests/application-server-integration-tests/../simple-webapp/target/simple-webapp.war
2020-11-29 19:36:53,992 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Check file /home/admin/Documents/Projects/apm/apm-agent-java/integration-tests/application-server-integration-tests/../cdi-app/cdi-app-standalone/target/cdi-app.war
2020-11-29 19:36:53,993 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Check file /home/admin/Documents/Projects/apm/apm-agent-java/integration-tests/application-server-integration-tests/../jsf-app/jsf-app-standalone/target/jsf-http-get.war
[tomcat] 29-Nov-2020 13:36:55.717 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Server version name:   Apache Tomcat/10.0.0-M10
[tomcat] 29-Nov-2020 13:36:55.721 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Server built:          Nov 12 2020 11:01:14 UTC
[tomcat] 29-Nov-2020 13:36:55.721 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Server version number: 10.0.0.0
[tomcat] 29-Nov-2020 13:36:55.721 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log OS Name:               Linux
[tomcat] 29-Nov-2020 13:36:55.721 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log OS Version:            5.3.0-64-generic
[tomcat] 29-Nov-2020 13:36:55.722 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Architecture:          amd64
[tomcat] 29-Nov-2020 13:36:55.722 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Java Home:             /usr/local/openjdk-8/jre
[tomcat] 29-Nov-2020 13:36:55.722 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log JVM Version:           1.8.0_275-b01
[tomcat] 29-Nov-2020 13:36:55.722 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log JVM Vendor:            Oracle Corporation
[tomcat] 29-Nov-2020 13:36:55.722 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log CATALINA_BASE:         /usr/local/tomcat
[tomcat] 29-Nov-2020 13:36:55.722 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log CATALINA_HOME:         /usr/local/tomcat
[tomcat] 29-Nov-2020 13:36:55.726 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.util.logging.config.file=/usr/local/tomcat/conf/logging.properties
[tomcat] 29-Nov-2020 13:36:55.726 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager
[tomcat] 29-Nov-2020 13:36:55.726 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djdk.tls.ephemeralDHKeySize=2048
[tomcat] 29-Nov-2020 13:36:55.727 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.protocol.handler.pkgs=org.apache.catalina.webresources
[tomcat] 29-Nov-2020 13:36:55.727 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dorg.apache.catalina.security.SecurityListener.UMASK=0027
[tomcat] 29-Nov-2020 13:36:55.727 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dignore.endorsed.dirs=
[tomcat] 29-Nov-2020 13:36:55.727 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dcatalina.base=/usr/local/tomcat
[tomcat] 29-Nov-2020 13:36:55.727 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Dcatalina.home=/usr/local/tomcat
[tomcat] 29-Nov-2020 13:36:55.727 INFO [main] org.apache.catalina.startup.VersionLoggerListener.log Command line argument: -Djava.io.tmpdir=/usr/local/tomcat/temp
[tomcat] 29-Nov-2020 13:36:55.737 INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent Loaded Apache Tomcat Native library [1.2.25] using APR version [1.6.5].
[tomcat] 29-Nov-2020 13:36:55.737 INFO [main] org.apache.catalina.core.AprLifecycleListener.lifecycleEvent APR capabilities: IPv6 [true], sendfile [true], accept filters [false], random [true].
[tomcat] 29-Nov-2020 13:36:55.740 INFO [main] org.apache.catalina.core.AprLifecycleListener.initializeSSL OpenSSL successfully initialized [OpenSSL 1.1.1d  10 Sep 2019]
[tomcat] 29-Nov-2020 13:36:56.110 INFO [main] org.apache.coyote.AbstractProtocol.init Initializing ProtocolHandler ["http-nio-8080"]
[tomcat] 29-Nov-2020 13:36:56.136 INFO [main] org.apache.catalina.startup.Catalina.load Server initialization in [622] milliseconds
[tomcat] 29-Nov-2020 13:36:56.163 INFO [main] org.apache.catalina.core.StandardService.startInternal Starting service [Catalina]
[tomcat] 29-Nov-2020 13:36:56.164 INFO [main] org.apache.catalina.core.StandardEngine.startInternal Starting Servlet engine: [Apache Tomcat/10.0.0-M10]
[tomcat] 29-Nov-2020 13:36:56.181 INFO [main] org.apache.catalina.startup.HostConfig.deployWAR Deploying web application archive [/usr/local/tomcat/webapps/jsf-http-get.war]
[tomcat] 29-Nov-2020 13:36:56.834 INFO [main] org.apache.jasper.servlet.TldScanner.scanJars At least one JAR was scanned for TLDs yet contained no TLDs. Enable debug logging for this logger for a complete list of JARs that were scanned but no TLDs were found in them. Skipping unneeded JARs during scanning can improve startup time and JSP compilation time.
[tomcat] 29-Nov-2020 13:36:56.858 SEVERE [main] org.apache.catalina.core.StandardContext.startInternal One or more listeners failed to start. Full details will be found in the appropriate container log file
[tomcat] 29-Nov-2020 13:36:56.869 SEVERE [main] org.apache.catalina.core.StandardContext.startInternal Context [/jsf-http-get] startup failed due to previous errors
[tomcat] 29-Nov-2020 13:36:56.881 INFO [main] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/usr/local/tomcat/webapps/jsf-http-get.war] has finished in [699] ms
[tomcat] 29-Nov-2020 13:36:56.881 INFO [main] org.apache.catalina.startup.HostConfig.deployWAR Deploying web application archive [/usr/local/tomcat/webapps/simple-webapp.war]
[tomcat] 29-Nov-2020 13:36:56.937 INFO [main] org.apache.jasper.servlet.TldScanner.scanJars At least one JAR was scanned for TLDs yet contained no TLDs. Enable debug logging for this logger for a complete list of JARs that were scanned but no TLDs were found in them. Skipping unneeded JARs during scanning can improve startup time and JSP compilation time.
[tomcat] 29-Nov-2020 13:36:56.958 INFO [main] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/usr/local/tomcat/webapps/simple-webapp.war] has finished in [77] ms
[tomcat] 29-Nov-2020 13:36:56.960 INFO [main] org.apache.catalina.startup.HostConfig.deployWAR Deploying web application archive [/usr/local/tomcat/webapps/cdi-app.war]
[tomcat] 29-Nov-2020 13:36:57.160 INFO [main] org.apache.jasper.servlet.TldScanner.scanJars At least one JAR was scanned for TLDs yet contained no TLDs. Enable debug logging for this logger for a complete list of JARs that were scanned but no TLDs were found in them. Skipping unneeded JARs during scanning can improve startup time and JSP compilation time.
[tomcat] 29-Nov-2020 13:36:57.163 INFO [main] org.apache.catalina.startup.HostConfig.deployWAR Deployment of web application archive [/usr/local/tomcat/webapps/cdi-app.war] has finished in [203] ms
[tomcat] 29-Nov-2020 13:36:57.177 INFO [main] org.apache.coyote.AbstractProtocol.start Starting ProtocolHandler ["http-nio-8080"]
[tomcat] 29-Nov-2020 13:36:57.194 INFO [main] org.apache.catalina.startup.Catalina.start Server startup in [1058] milliseconds
[tomcat] 2020-11-29 13:36:57,564 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-configuration-reloader` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:57,579 [Attach Listener] DEBUG co.elastic.apm.agent.impl.payload.SystemInfo - container ID is 79e03f91e79a66092d281de324818fe094830ac32e0bcee4731eb2f0383160f5
[tomcat] 2020-11-29 13:36:57,600 [Attach Listener] INFO  co.elastic.apm.agent.util.JmxUtils - Found JVM-specific OperatingSystemMXBean interface: com.sun.management.OperatingSystemMXBean
[tomcat] 2020-11-29 13:36:57,637 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-log-shipper` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:57,638 [Attach Listener] INFO  co.elastic.apm.agent.configuration.StartupInfo - Starting Elastic APM 1.19.1-SNAPSHOT.bf1d229 as tomcat-application on Java 1.8.0_275 Runtime version: 1.8.0_275-b01 VM version: 25.275-b01 (Oracle Corporation) Linux 5.3.0-64-generic
[tomcat] 2020-11-29 13:36:57,638 [Attach Listener] INFO  co.elastic.apm.agent.configuration.StartupInfo - VM Arguments: [-Djava.util.logging.config.file=/usr/local/tomcat/conf/logging.properties, -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager, -Djdk.tls.ephemeralDHKeySize=2048, -Djava.protocol.handler.pkgs=org.apache.catalina.webresources, -Dorg.apache.catalina.security.SecurityListener.UMASK=0027, -Dignore.endorsed.dirs=, -Dcatalina.base=/usr/local/tomcat, -Dcatalina.home=/usr/local/tomcat, -Djava.io.tmpdir=/usr/local/tomcat/temp]
[tomcat] 2020-11-29 13:36:57,639 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - disable_instrumentations: '' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,639 [Attach Listener] WARN  co.elastic.apm.agent.configuration.StartupInfo - Detected usage of an old configuration key: 'disabled_instrumentations'. Please use 'disable_instrumentations' instead.
[tomcat] 2020-11-29 13:36:57,639 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - capture_body: 'ALL' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,639 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - trace_methods: 'public @@javax.enterprise.context.NormalScope co.elastic.*' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,639 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - server_urls: 'http://apm-server:1080' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,640 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - report_sync: 'true' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,640 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - metrics_interval: '1s' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,640 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - application_packages: 'co.elastic' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,640 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - log_level: 'DEBUG' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,640 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - transaction_ignore_urls: '/status*,/favicon.ico' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,641 [Attach Listener] WARN  co.elastic.apm.agent.configuration.StartupInfo - Detected usage of an old configuration key: 'ignore_urls'. Please use 'transaction_ignore_urls' instead.
[tomcat] 2020-11-29 13:36:57,641 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - circuit_breaker_enabled: 'true' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,641 [Attach Listener] DEBUG co.elastic.apm.agent.configuration.StartupInfo - capture_jmx_metrics: 'object_name[java.lang:type=Memory] attribute[HeapMemoryUsage:metric_name=test_heap_metric]' (source: Environment Variables)
[tomcat] 2020-11-29 13:36:57,642 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - No plugins dir
[tomcat] 2020-11-29 13:36:57,661 [Attach Listener] DEBUG co.elastic.apm.agent.util.DependencyInjectingServiceLoader - unable to instantiate 'co.elastic.apm.agent.httpclient.HttpClientInstrumentation', unsupported class version error: co/elastic/apm/agent/httpclient/HttpClientInstrumentation has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
[tomcat] 2020-11-29 13:36:57,661 [Attach Listener] DEBUG co.elastic.apm.agent.util.DependencyInjectingServiceLoader - unable to instantiate 'co.elastic.apm.agent.httpclient.HttpClientAsyncInstrumentation', unsupported class version error: co/elastic/apm/agent/httpclient/HttpClientAsyncInstrumentation has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
[tomcat] 2020-11-29 13:36:57,662 [Attach Listener] DEBUG co.elastic.apm.agent.util.DependencyInjectingServiceLoader - unable to instantiate 'co.elastic.apm.agent.httpclient.HttpRequestHeadersInstrumentation', unsupported class version error: co/elastic/apm/agent/httpclient/HttpRequestHeadersInstrumentation has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
[tomcat] 2020-11-29 13:36:57,701 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-type-cache-pool-cleaner` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:57,703 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.httpclient.ApacheHttpClientInstrumentation
[tomcat] 2020-11-29 13:36:57,738 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.httpclient.ApacheHttpAsyncClientInstrumentation
[tomcat] 2020-11-29 13:36:57,744 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.httpclient.ApacheHttpAsyncClientRedirectInstrumentation
[tomcat] 2020-11-29 13:36:57,747 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.httpclient.LegacyApacheHttpClientInstrumentation
[tomcat] 2020-11-29 13:36:57,751 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.ElasticApmApiInstrumentation$StartTransactionInstrumentation
[tomcat] 2020-11-29 13:36:57,754 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.ElasticApmApiInstrumentation$StartTransactionWithRemoteParentInstrumentation
[tomcat] 2020-11-29 13:36:57,758 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.ElasticApmApiInstrumentation$CurrentTransactionInstrumentation
[tomcat] 2020-11-29 13:36:57,761 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.ElasticApmApiInstrumentation$CurrentSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,764 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.ElasticApmApiInstrumentation$CaptureExceptionInstrumentation
[tomcat] 2020-11-29 13:36:57,766 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.TransactionInstrumentation$SetUserInstrumentation
[tomcat] 2020-11-29 13:36:57,769 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.TransactionInstrumentation$EnsureParentIdInstrumentation
[tomcat] 2020-11-29 13:36:57,772 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.TransactionInstrumentation$SetResultInstrumentation
[tomcat] 2020-11-29 13:36:57,774 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.TransactionInstrumentation$AddCustomContextInstrumentation
[tomcat] 2020-11-29 13:36:57,776 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$SetNameInstrumentation
[tomcat] 2020-11-29 13:36:57,778 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$SetTypeInstrumentation
[tomcat] 2020-11-29 13:36:57,780 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$SetTypesInstrumentation
[tomcat] 2020-11-29 13:36:57,782 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$DoCreateSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,785 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$SetStartTimestampInstrumentation
[tomcat] 2020-11-29 13:36:57,787 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$EndInstrumentation
[tomcat] 2020-11-29 13:36:57,789 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$EndWithTimestampInstrumentation
[tomcat] 2020-11-29 13:36:57,791 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$CaptureExceptionInstrumentation
[tomcat] 2020-11-29 13:36:57,793 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$GetIdInstrumentation
[tomcat] 2020-11-29 13:36:57,796 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$GetTraceIdInstrumentation
[tomcat] 2020-11-29 13:36:57,798 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$AddStringLabelInstrumentation
[tomcat] 2020-11-29 13:36:57,801 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$AddNumberLabelInstrumentation
[tomcat] 2020-11-29 13:36:57,802 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$AddBooleanLabelInstrumentation
[tomcat] 2020-11-29 13:36:57,804 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$ActivateInstrumentation
[tomcat] 2020-11-29 13:36:57,807 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$IsSampledInstrumentation
[tomcat] 2020-11-29 13:36:57,809 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$InjectTraceHeadersInstrumentation
[tomcat] 2020-11-29 13:36:57,811 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.CaptureExceptionInstrumentation
[tomcat] 2020-11-29 13:36:57,813 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.ApiScopeInstrumentation
[tomcat] 2020-11-29 13:36:57,815 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.CaptureTransactionInstrumentation
[tomcat] 2020-11-29 13:36:57,819 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.CaptureSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,822 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.TracedInstrumentation
[tomcat] 2020-11-29 13:36:57,825 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.AbstractSpanInstrumentation$LegacyCaptureExceptionInstrumentation
[tomcat] 2020-11-29 13:36:57,827 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$SetNameInstrumentation
[tomcat] 2020-11-29 13:36:57,828 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$SetTypeInstrumentation
[tomcat] 2020-11-29 13:36:57,830 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$DoCreateSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,833 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$EndInstrumentation
[tomcat] 2020-11-29 13:36:57,834 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$CaptureExceptionInstrumentation
[tomcat] 2020-11-29 13:36:57,836 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$GetIdInstrumentation
[tomcat] 2020-11-29 13:36:57,838 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$GetTraceIdInstrumentation
[tomcat] 2020-11-29 13:36:57,840 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$AddTagInstrumentation
[tomcat] 2020-11-29 13:36:57,842 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$ActivateInstrumentation
[tomcat] 2020-11-29 13:36:57,843 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.pluginapi.LegacySpanInstrumentation$IsSampledInstrumentation
[tomcat] 2020-11-29 13:36:57,845 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.asynchttpclient.AbstractAsyncHttpClientInstrumentation$AsyncHttpClientInstrumentation
[tomcat] 2020-11-29 13:36:57,847 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.bootdelegation.BootstrapDelegationClassLoaderInstrumentation
[tomcat] 2020-11-29 13:36:57,848 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.errorlogging.Slf4jLoggerErrorCapturingInstrumentation
[tomcat] 2020-11-29 13:36:57,851 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.errorlogging.Log4j2LoggerErrorCapturingInstrumentation
[tomcat] 2020-11-29 13:36:57,854 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.grails.GrailsTransactionNameInstrumentation
[tomcat] 2020-11-29 13:36:57,856 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.concurrent.ExecutorInstrumentation$ExecutorRunnableInstrumentation
[tomcat] 2020-11-29 13:36:57,860 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.concurrent.ExecutorInstrumentation$ExecutorCallableInstrumentation
[tomcat] 2020-11-29 13:36:57,864 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.concurrent.ExecutorInstrumentation$ExecutorInvokeAnyAllInstrumentation
[tomcat] 2020-11-29 13:36:57,870 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.concurrent.ExecutorInstrumentation$ForkJoinPoolInstrumentation
[tomcat] 2020-11-29 13:36:57,874 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.concurrent.ForkJoinTaskInstrumentation
[tomcat] 2020-11-29 13:36:57,877 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jaxrs.JaxRsTransactionNameInstrumentation
[tomcat] 2020-11-29 13:36:57,878 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jaxws.JaxWsTransactionNameInstrumentation
[tomcat] 2020-11-29 13:36:57,879 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jdbc.ConnectionInstrumentation
[tomcat] 2020-11-29 13:36:57,881 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecuteWithQueryInstrumentation
[tomcat] 2020-11-29 13:36:57,884 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecuteUpdateWithQueryInstrumentation
[tomcat] 2020-11-29 13:36:57,886 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecuteUpdateNoQueryInstrumentation
[tomcat] 2020-11-29 13:36:57,889 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jdbc.StatementInstrumentation$AddBatchInstrumentation
[tomcat] 2020-11-29 13:36:57,890 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecuteBatchInstrumentation
[tomcat] 2020-11-29 13:36:57,893 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecutePreparedStatementInstrumentation
[tomcat] 2020-11-29 13:36:57,895 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.jedis.JedisInstrumentation
[tomcat] 2020-11-29 13:36:57,896 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.jedis.JedisSpanNameInstrumentation
[tomcat] 2020-11-29 13:36:57,896 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce34StartSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,897 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce34StopSpanInstrumentation$OnComplete
[tomcat] 2020-11-29 13:36:57,897 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce34StopSpanInstrumentation$OnCompleteExceptionally
[tomcat] 2020-11-29 13:36:57,897 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce34StopSpanInstrumentation$OnCancel
[tomcat] 2020-11-29 13:36:57,897 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce5StartSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,898 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce5StopSpanInstrumentation$OnComplete
[tomcat] 2020-11-29 13:36:57,898 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce5StopSpanInstrumentation$OnCompleteExceptionally
[tomcat] 2020-11-29 13:36:57,898 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.lettuce.Lettuce5StopSpanInstrumentation$OnCancel
[tomcat] 2020-11-29 13:36:57,898 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.redis.redisson.RedisConnectionInstrumentation
[tomcat] 2020-11-29 13:36:57,898 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jms.JmsMessageProducerInstrumentation$JmsMessageProducerNoDestinationInstrumentation
[tomcat] 2020-11-29 13:36:57,899 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jms.JmsMessageProducerInstrumentation$JmsMessageProducerWithDestinationInstrumentation
[tomcat] 2020-11-29 13:36:57,899 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jms.JmsMessageConsumerInstrumentation$ReceiveInstrumentation
[tomcat] 2020-11-29 13:36:57,899 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jms.JmsMessageConsumerInstrumentation$SetMessageListenerInstrumentation
[tomcat] 2020-11-29 13:36:57,899 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jms.JmsMessageListenerInstrumentation
[tomcat] 2020-11-29 13:36:57,900 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jsf.JsfLifecycleInstrumentation$JsfLifecycleExecuteInstrumentation
[tomcat] 2020-11-29 13:36:57,902 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.jsf.JsfLifecycleInstrumentation$JsfLifecycleRenderInstrumentation
[tomcat] 2020-11-29 13:36:57,905 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.hibernatesearch.HibernateSearch5Instrumentation
[tomcat] 2020-11-29 13:36:57,907 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.hibernatesearch.HibernateSearch6Instrumentation
[tomcat] 2020-11-29 13:36:57,910 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.es.restclient.v5_6.ElasticsearchClientSyncInstrumentation
[tomcat] 2020-11-29 13:36:57,910 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.es.restclient.v5_6.ElasticsearchClientAsyncInstrumentation
[tomcat] 2020-11-29 13:36:57,910 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.es.restclient.v6_4.ElasticsearchClientSyncInstrumentation
[tomcat] 2020-11-29 13:36:57,911 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.es.restclient.v6_4.ElasticsearchClientAsyncInstrumentation
[tomcat] 2020-11-29 13:36:57,911 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.grpc.ServerCallListenerInstrumentation$FinalMethodCall
[tomcat] 2020-11-29 13:36:57,913 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.grpc.ServerCallListenerInstrumentation$NonFinalMethodCall
[tomcat] 2020-11-29 13:36:57,916 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.grpc.ServerCallHandlerInstrumentation
[tomcat] 2020-11-29 13:36:57,918 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.grpc.ServerCallInstrumentation
[tomcat] 2020-11-29 13:36:57,920 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.grpc.ChannelInstrumentation
[tomcat] 2020-11-29 13:36:57,922 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.grpc.ClientCallImplInstrumentation$Start
[tomcat] 2020-11-29 13:36:57,925 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.kafka.KafkaProducerInstrumentation
[tomcat] 2020-11-29 13:36:57,925 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.kafka.KafkaConsumerInstrumentation
[tomcat] 2020-11-29 13:36:57,926 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.kafka.KafkaProducerHeadersInstrumentation
[tomcat] 2020-11-29 13:36:57,926 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.kafka.ConsumerRecordsIteratorInstrumentation
[tomcat] 2020-11-29 13:36:57,926 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.kafka.ConsumerRecordsRecordsInstrumentation
[tomcat] 2020-11-29 13:36:57,926 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.kafka.ConsumerRecordsRecordListInstrumentation
[tomcat] 2020-11-29 13:36:57,927 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.mongoclient.ConnectionCommandInstrumentation
[tomcat] 2020-11-29 13:36:57,927 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.mongoclient.ConnectionInstrumentation
[tomcat] 2020-11-29 13:36:57,927 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.okhttp.OkHttpClientInstrumentation
[tomcat] 2020-11-29 13:36:57,930 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.okhttp.OkHttp3ClientInstrumentation
[tomcat] 2020-11-29 13:36:57,933 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.okhttp.OkHttpClientAsyncInstrumentation
[tomcat] 2020-11-29 13:36:57,936 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.okhttp.OkHttp3ClientAsyncInstrumentation
[tomcat] 2020-11-29 13:36:57,940 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ApmSpanInstrumentation$FinishInstrumentation
[tomcat] 2020-11-29 13:36:57,940 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ApmSpanInstrumentation$LogInstrumentation
[tomcat] 2020-11-29 13:36:57,940 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ApmSpanInstrumentation$TagInstrumentation
[tomcat] 2020-11-29 13:36:57,940 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ApmSpanInstrumentation$SetOperationName
[tomcat] 2020-11-29 13:36:57,941 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ApmSpanInstrumentation$GetTraceContextInstrumentation
[tomcat] 2020-11-29 13:36:57,941 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ApmSpanBuilderInstrumentation$CreateSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,941 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ScopeManagerInstrumentation$ActivateInstrumentation
[tomcat] 2020-11-29 13:36:57,941 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ScopeManagerInstrumentation$CurrentSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,941 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ScopeManagerInstrumentation$CurrentTraceContextInstrumentation
[tomcat] 2020-11-29 13:36:57,941 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ApmScopeInstrumentation
[tomcat] 2020-11-29 13:36:57,941 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.SpanContextInstrumentation$BaggageItemsInstrumentation
[tomcat] 2020-11-29 13:36:57,942 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.SpanContextInstrumentation$ToTraceIdInstrumentation
[tomcat] 2020-11-29 13:36:57,942 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.SpanContextInstrumentation$ToSpanIdInstrumentation
[tomcat] 2020-11-29 13:36:57,942 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ExternalSpanContextInstrumentation$ToTraceIdInstrumentation
[tomcat] 2020-11-29 13:36:57,942 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ExternalSpanContextInstrumentation$ToSpanIdInstrumentation
[tomcat] 2020-11-29 13:36:57,942 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.opentracing.impl.ElasticApmTracerInstrumentation
[tomcat] 2020-11-29 13:36:57,942 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.servlet.ServletInstrumentation
[tomcat] 2020-11-29 13:36:57,946 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.servlet.ServletVersionInstrumentation$Init
[tomcat] 2020-11-29 13:36:57,947 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.servlet.ServletVersionInstrumentation$Service
[tomcat] 2020-11-29 13:36:57,949 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.servlet.FilterChainInstrumentation
[tomcat] 2020-11-29 13:36:57,952 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.servlet.AsyncInstrumentation$StartAsyncInstrumentation
[tomcat] 2020-11-29 13:36:57,954 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.servlet.AsyncInstrumentation$AsyncContextInstrumentation
[tomcat] 2020-11-29 13:36:57,957 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.servlet.RequestStreamRecordingInstrumentation
[tomcat] 2020-11-29 13:36:57,961 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.micrometer.MicrometerInstrumentation
[tomcat] 2020-11-29 13:36:57,963 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.resttemplate.SpringRestTemplateInstrumentation
[tomcat] 2020-11-29 13:36:57,966 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.springwebmvc.SpringTransactionNameInstrumentation
[tomcat] 2020-11-29 13:36:57,968 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.springwebmvc.ServletWrappingControllerServiceNameInstrumentation
[tomcat] 2020-11-29 13:36:57,970 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.springwebmvc.ViewRenderInstrumentation
[tomcat] 2020-11-29 13:36:57,973 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.springwebmvc.SpringServiceNameInstrumentation
[tomcat] 2020-11-29 13:36:57,975 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.springwebmvc.ExceptionHandlerInstrumentation
[tomcat] 2020-11-29 13:36:57,977 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.urlconnection.HttpUrlConnectionInstrumentation$CreateSpanInstrumentation
[tomcat] 2020-11-29 13:36:57,980 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.urlconnection.HttpUrlConnectionInstrumentation$DisconnectInstrumentation
[tomcat] 2020-11-29 13:36:57,983 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.urlconnection.SSLContextInstrumentation
[tomcat] 2020-11-29 13:36:57,985 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.scheduled.ScheduledTransactionNameInstrumentation
[tomcat] 2020-11-29 13:36:57,988 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.scheduled.TimerTaskInstrumentation
[tomcat] 2020-11-29 13:36:57,991 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.quartzjob.JobTransactionNameInstrumentation
[tomcat] 2020-11-29 13:36:57,994 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.process.ProcessStartInstrumentation
[tomcat] 2020-11-29 13:36:57,996 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.process.ProcessExitInstrumentation$WaitFor
[tomcat] 2020-11-29 13:36:57,998 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.process.ProcessExitInstrumentation$Destroy
[tomcat] 2020-11-29 13:36:58,000 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.process.CommonsExecAsyncInstrumentation
[tomcat] 2020-11-29 13:36:58,003 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.dubbo.ApacheMonitorFilterInstrumentation
[tomcat] 2020-11-29 13:36:58,004 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.dubbo.AlibabaMonitorFilterInstrumentation
[tomcat] 2020-11-29 13:36:58,004 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.dubbo.AlibabaRpcContextInstrumentation
[tomcat] 2020-11-29 13:36:58,004 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.scalaconcurrent.FutureInstrumentation$ConstructorInstrumentation
[tomcat] 2020-11-29 13:36:58,006 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.scalaconcurrent.FutureInstrumentation$RunInstrumentation
[tomcat] 2020-11-29 13:36:58,009 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applying instrumentation co.elastic.apm.agent.bci.methodmatching.TraceMethodInstrumentation
[tomcat] 2020-11-29 13:36:58,009 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Applied 152 advices
[tomcat] 2020-11-29 13:36:58,103 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class org.apache.catalina.loader.ParallelWebappClassLoader
[tomcat] 2020-11-29 13:36:58,103 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class org.apache.catalina.loader.WebappClassLoaderBase
[tomcat] 2020-11-29 13:36:58,167 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ForkJoinTaskInstrumentation: (is(class java.util.concurrent.ForkJoinTask) and not(isInterface())) matches class java.util.concurrent.ForkJoinTask
[tomcat] 2020-11-29 13:36:58,168 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ForkJoinPool
[tomcat] 2020-11-29 13:36:58,171 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class co.elastic.apm.agent.bci.classloading.ExternalPluginClassLoader
[tomcat] 2020-11-29 13:36:58,249 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$NamedDaemonThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,264 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$1
[tomcat] 2020-11-29 13:36:58,305 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ScheduledThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,323 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation CreateSpanInstrumentation: ((hasSuperType(erasure(is(class java.net.HttpURLConnection))) and not(name(equals(sun.net.www.protocol.https.HttpsURLConnectionImpl)))) and not(isInterface())) matches class java.net.HttpURLConnection
[tomcat] 2020-11-29 13:36:58,363 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,363 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$FinalizableDelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,363 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$DelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,364 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.AbstractExecutorService
[tomcat] 2020-11-29 13:36:58,393 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class org.apache.catalina.loader.ParallelWebappClassLoader
[tomcat] 2020-11-29 13:36:58,428 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class org.apache.catalina.loader.WebappClassLoaderBase
[tomcat] 2020-11-29 13:36:58,462 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation BootstrapDelegationClassLoaderInstrumentation: (((name(equals(loadClass)) and returns(erasure(is(class java.lang.Class)))) and (hasParameter(hasTypes(erasures(containing(is(class java.lang.String))))) or hasParameter(hasTypes(erasures(containing(is(class java.lang.String), is(boolean))))))) and not(isAbstract())) matches public java.lang.Class org.apache.catalina.loader.WebappClassLoaderBase.loadClass(java.lang.String,boolean) throws java.lang.ClassNotFoundException
[tomcat] 2020-11-29 13:36:58,477 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation BootstrapDelegationClassLoaderInstrumentation: (((name(equals(loadClass)) and returns(erasure(is(class java.lang.Class)))) and (hasParameter(hasTypes(erasures(containing(is(class java.lang.String))))) or hasParameter(hasTypes(erasures(containing(is(class java.lang.String), is(boolean))))))) and not(isAbstract())) matches public java.lang.Class org.apache.catalina.loader.WebappClassLoaderBase.loadClass(java.lang.String) throws java.lang.ClassNotFoundException
[tomcat] 2020-11-29 13:36:58,489 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ForkJoinTaskInstrumentation: (is(class java.util.concurrent.ForkJoinTask) and not(isInterface())) matches class java.util.concurrent.ForkJoinTask
[tomcat] 2020-11-29 13:36:58,500 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ForkJoinTaskInstrumentation: ((name(equals(fork)) and returns(erasure(is(class java.util.concurrent.ForkJoinTask)))) and not(isAbstract())) matches public final java.util.concurrent.ForkJoinTask java.util.concurrent.ForkJoinTask.fork()
[tomcat] 2020-11-29 13:36:58,507 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ForkJoinPool
[tomcat] 2020-11-29 13:36:58,507 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ForkJoinPool
[tomcat] 2020-11-29 13:36:58,508 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ForkJoinPool
[tomcat] 2020-11-29 13:36:58,508 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ForkJoinPoolInstrumentation: ((hasSuperType(erasure(is(class java.util.concurrent.ForkJoinPool))) and ((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$)))))) and not(isInterface())) matches class java.util.concurrent.ForkJoinPool
[tomcat] 2020-11-29 13:36:58,535 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.util.List java.util.concurrent.ForkJoinPool.invokeAll(java.util.Collection)
[tomcat] 2020-11-29 13:36:58,540 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ForkJoinPoolInstrumentation: (((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask)))))) or ((name(equals(submit)) and returns(erasure(is(class java.util.concurrent.ForkJoinTask)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask))))))) or ((name(equals(invoke)) and returns(erasure(is(class java.lang.Object)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask))))))) and not(isAbstract())) matches public java.util.concurrent.ForkJoinTask java.util.concurrent.ForkJoinPool.submit(java.util.concurrent.ForkJoinTask)
[tomcat] 2020-11-29 13:36:58,542 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorCallableInstrumentation: ((((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable)))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.ForkJoinTask java.util.concurrent.ForkJoinPool.submit(java.util.concurrent.Callable)
[tomcat] 2020-11-29 13:36:58,543 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] ForkJoinTask java.util.concurrent.ForkJoinPool.submit(java.lang.Runnable,java.lang.Object)
[tomcat] 2020-11-29 13:36:58,545 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] ForkJoinTask java.util.concurrent.ForkJoinPool.submit(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,546 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ForkJoinPoolInstrumentation: (((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask)))))) or ((name(equals(submit)) and returns(erasure(is(class java.util.concurrent.ForkJoinTask)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask))))))) or ((name(equals(invoke)) and returns(erasure(is(class java.lang.Object)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask))))))) and not(isAbstract())) matches public java.lang.Object java.util.concurrent.ForkJoinPool.invoke(java.util.concurrent.ForkJoinTask)
[tomcat] 2020-11-29 13:36:58,549 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ForkJoinPoolInstrumentation: (((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask)))))) or ((name(equals(submit)) and returns(erasure(is(class java.util.concurrent.ForkJoinTask)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask))))))) or ((name(equals(invoke)) and returns(erasure(is(class java.lang.Object)))) and hasParameter(hasTypes(erasures(containing(is(class java.util.concurrent.ForkJoinTask))))))) and not(isAbstract())) matches public void java.util.concurrent.ForkJoinPool.execute(java.util.concurrent.ForkJoinTask)
[tomcat] 2020-11-29 13:36:58,550 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public void java.util.concur
[tomcat] rent.ForkJoinPool.execute(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,556 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class co.elastic.apm.agent.bci.classloading.ExternalPluginClassLoader
[tomcat] 2020-11-29 13:36:58,559 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$NamedDaemonThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,559 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$NamedDaemonThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,560 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$NamedDaemonThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,568 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$1
[tomcat] 2020-11-29 13:36:58,569 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$1
[tomcat] 2020-11-29 13:36:58,569 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class co.elastic.apm.agent.util.ExecutorUtils$1
[tomcat] 2020-11-29 13:36:58,579 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ScheduledThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,579 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ScheduledThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,579 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ScheduledThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,590 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorCallableInstrumentation: ((((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable)))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.ScheduledFuture java.util.concurrent.ScheduledThreadPoolExecutor.schedule(java.util.concurrent.Callable,long,java.util.concurrent.TimeUnit)
[tomcat] 2020-11-29 13:36:58,592 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] ScheduledFuture java.util.concurrent.ScheduledThreadPoolExecutor.schedule(java.lang.Runnable,long,java.util.concurrent.TimeUnit)
[tomcat] 2020-11-29 13:36:58,593 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorCallableInstrumentation: ((((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable)))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.Future java.util.concurrent.ScheduledThreadPoolExecutor.submit(java.util.concurrent.Callable)
[tomcat] 2020-11-29 13:36:58,594 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] Future java.util.concurrent.ScheduledThreadPoolExecutor.submit(java.lang.Runnable,java.lang.Object)
[tomcat] 2020-11-29 13:36:58,595 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] Future java.util.concurrent.ScheduledThreadPoolExecutor.submit(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,597 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public void java.util.concur
[tomcat] rent.ScheduledThreadPoolExecutor.execute(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,599 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation CreateSpanInstrumentation: ((hasSuperType(erasure(is(class java.net.HttpURLConnection))) and not(name(equals(sun.net.www.protocol.https.HttpsURLConnectionImpl)))) and not(isInterface())) matches class java.net.HttpURLConnection
[tomcat] 2020-11-29 13:36:58,599 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation DisconnectInstrumentation: ((hasSuperType(erasure(is(class java.net.HttpURLConnection))) and not(name(equals(sun.net.www.protocol.https.HttpsURLConnectionImpl)))) and not(isInterface())) matches class java.net.HttpURLConnection
[tomcat] 2020-11-29 13:36:58,610 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,611 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,611 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.ThreadPoolExecutor
[tomcat] 2020-11-29 13:36:58,630 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public void java.util.concur
[tomcat] rent.ThreadPoolExecutor.execute(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,633 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$FinalizableDelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,633 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$FinalizableDelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,634 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$FinalizableDelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,641 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$DelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,641 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$DelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,641 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.Executors$DelegatedExecutorService
[tomcat] 2020-11-29 13:36:58,649 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.util.List java.util.concurrent.Executors$DelegatedExecutorService.invokeAll(java.util.Collection) throws java.lang.InterruptedException
[tomcat] 2020-11-29 13:36:58,651 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.util.List java.util.concurrent.Executors$DelegatedExecutorService.invokeAll(java.util.Collection,long,java.util.concurrent.TimeUnit) throws java.lang.InterruptedException
[tomcat] 2020-11-29 13:36:58,652 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.lang.Object java.util.concurrent.Executors$DelegatedExecutorService.invokeAny(java.util.Collection,long,java.util.concurrent.TimeUnit) throws java.lang.InterruptedException,java.util.concurrent.ExecutionException,java.util.concurrent.TimeoutException
[tomcat] 2020-11-29 13:36:58,654 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.lang.Object java.util.concurrent.Executors$DelegatedExecutorService.invokeAny(java.util.Collection) throws java.lang.InterruptedException,java.util.concurrent.ExecutionException
[tomcat] 2020-11-29 13:36:58,655 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] Future java.util.concurrent.Executors$DelegatedExecutorService.submit(java.lang.Runnable,java.lang.Object)
[tomcat] 2020-11-29 13:36:58,656 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorCallableInstrumentation: ((((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable)))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.Future java.util.concurrent.Executors$DelegatedExecutorService.submit(java.util.concurrent.Callable)
[tomcat] 2020-11-29 13:36:58,657 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] Future java.util.concurrent.Executors$DelegatedExecutorService.submit(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,658 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public void java.util.concur
[tomcat] rent.Executors$DelegatedExecutorService.execute(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,659 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorRunnableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.AbstractExecutorService
[tomcat] 2020-11-29 13:36:58,660 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorCallableInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.AbstractExecutorService
[tomcat] 2020-11-29 13:36:58,660 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((((((hasSuperType(erasure(name(equals(java.util.concurrent.Executor)))) and not(name(equals(org.apache.felix.resolver.ResolverImpl$DumbExecutor)))) and not(name(contains(jetty)))) and not(name(contains(tomcat)))) and not(name(contains(jboss)))) and not(name(contains(undertow)))) and not(name(contains(netty)))) and not(name(startsWith(com.hazelcast)))) and not((name(contains($Proxy)) or name(contains($$))))) and not(isInterface())) matches class java.util.concurrent.AbstractExecutorService
[tomcat] 2020-11-29 13:36:58,668 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.util.List java.util.concurrent.AbstractExecutorService.invokeAll(java.util.Collection,long,java.util.concurrent.TimeUnit) throws java.lang.InterruptedException
[tomcat] 2020-11-29 13:36:58,670 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.util.List java.util.concurrent.AbstractExecutorService.invokeAll(java.util.Collection) throws java.lang.InterruptedException
[tomcat] 2020-11-29 13:36:58,672 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.lang.Object java.util.concurrent.AbstractExecutorService.invokeAny(java.util.Collection) throws java.lang.InterruptedException,java.util.concurrent.ExecutionException
[tomcat] 2020-11-29 13:36:58,674 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorInvokeAnyAllInstrumentation: (((((name(startsWith(invoke)) and (name(endsWith(Any)) or name(endsWith(All)))) and isPublic()) and hasParameter(hasTypes(with(0 matches erasure(is(interface java.util.Collection)))))) and isOverriddenFrom(erasure(is(interface java.util.concurrent.ExecutorService)))) and not(isAbstract())) matches public java.lang.Object java.util.concurrent.AbstractExecutorService.invokeAny(java.util.Collection,long,java.util.concurrent.TimeUnit) throws java.lang.InterruptedException,java.util.concurrent.ExecutionException,java.util.concurrent.TimeoutException
[tomcat] 2020-11-29 13:36:58,675 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorCallableInstrumentation: ((((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable)))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.util.concurrent.Callable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.Future java.util.concurrent.AbstractExecutorService.submit(java.util.concurrent.Callable)
[tomcat] 2020-11-29 13:36:58,676 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] Future java.util.concurrent.AbstractExecutorService.submit(java.lang.Runnable)
[tomcat] 2020-11-29 13:36:58,678 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutorRunnableInstrumentation: ((((((name(equals(execute)) and returns(erasure(is(void)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable)))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable))))))) or ((name(equals(submit)) and returns(erasure(hasSuperType(erasure(is(interface java.util.concurrent.Future)))))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(class java.lang.Object))))))) or ((name(equals(schedule)) and returns(erasure(is(interface java.util.concurrent.ScheduledFuture)))) and hasParameter(hasTypes(erasures(containing(is(interface java.lang.Runnable), is(long), is(class java.util.concurrent.TimeUnit))))))) and not(isAbstract())) matches public java.util.concurrent.
[tomcat] Future java.util.concurrent.AbstractExecutorService.submit(java.lang.Runnable,java.lang.Object)
[tomcat] 2020-11-29 13:36:58,811 [Attach Listener] DEBUG co.elastic.apm.agent.bci.IndyPluginClassLoaderFactory - Creating plugin class loader for null containing [co.elastic.apm.agent.bci.classloading.LookupExposer]
[tomcat] 2020-11-29 13:36:58,814 [Attach Listener] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class co.elastic.apm.agent.bci.classloading.IndyPluginClassLoader
[tomcat] 2020-11-29 13:36:58,846 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-server-healthcheck` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:58,849 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-remote-config-poller` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:58,852 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation CreateSpanInstrumentation: ((hasSuperType(erasure(is(class java.net.HttpURLConnection))) and not(name(equals(sun.net.www.protocol.https.HttpsURLConnectionImpl)))) and not(isInterface())) matches class sun.net.www.protocol.http.HttpURLConnection
[tomcat] 2020-11-29 13:36:58,853 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation DisconnectInstrumentation: ((hasSuperType(erasure(is(class java.net.HttpURLConnection))) and not(name(equals(sun.net.www.protocol.https.HttpsURLConnectionImpl)))) and not(isInterface())) matches class sun.net.www.protocol.http.HttpURLConnection
[tomcat] 2020-11-29 13:36:58,886 [Attach Listener] DEBUG co.elastic.apm.agent.impl.circuitbreaker.GCStressMonitor - Registering a heap memory pool (PS Eden Space) for stress monitoring
[tomcat] 2020-11-29 13:36:58,887 [Attach Listener] DEBUG co.elastic.apm.agent.impl.circuitbreaker.GCStressMonitor - Registering a heap memory pool (PS Old Gen) for stress monitoring
[tomcat] 2020-11-29 13:36:58,888 [Attach Listener] DEBUG co.elastic.apm.agent.impl.circuitbreaker.SystemCpuStressMonitor - Successfully obtained reference to the getSystemCpuLoad method of this JVM's OperatingSystemMXBean implementation
[tomcat] 2020-11-29 13:36:58,890 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-circuit-breaker` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:58,891 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-weak-map-cleaner` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:58,891 [elastic-apm-circuit-breaker] DEBUG co.elastic.apm.agent.impl.circuitbreaker.GCStressMonitor - Heap PS Eden Space pool usage after the last GC is below the threshold of 0.95: 0/1361575936 (used/max)
[tomcat] 2020-11-29 13:36:58,892 [elastic-apm-circuit-breaker] DEBUG co.elastic.apm.agent.impl.circuitbreaker.GCStressMonitor - Heap PS Old Gen pool usage after the last GC is below the threshold of 0.95: 8146432/2779774976 (used/max)
[tomcat] 2020-11-29 13:36:58,893 [elastic-apm-circuit-breaker] DEBUG co.elastic.apm.agent.impl.circuitbreaker.SystemCpuStressMonitor - System CPU measurement: 0.8
[tomcat] 2020-11-29 13:36:58,894 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation CreateSpanInstrumentation: ((((name(equals(connect)) and hasParameter(ofSize(0))) or (name(equals(getOutputStream)) and hasParameter(ofSize(0)))) or (name(equals(getInputStream)) and hasParameter(ofSize(0)))) and not(isAbstract())) matches public void sun.net.www.protocol.http.HttpURLConnection.connect() throws java.io.IOException
[tomcat] 2020-11-29 13:36:58,894 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-shared` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:58,903 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation CreateSpanInstrumentation: ((((name(equals(connect)) and hasParameter(ofSize(0))) or (name(equals(getOutputStream)) and hasParameter(ofSize(0)))) or (name(equals(getInputStream)) and hasParameter(ofSize(0)))) and not(isAbstract())) matches public synchronized java.io.OutputStream sun.net.www.protocol.http.HttpURLConnection.getOutputStream() throws java.io.IOException
[tomcat] 2020-11-29 13:36:58,908 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation CreateSpanInstrumentation: ((((name(equals(connect)) and hasParameter(ofSize(0))) or (name(equals(getOutputStream)) and hasParameter(ofSize(0)))) or (name(equals(getInputStream)) and hasParameter(ofSize(0)))) and not(isAbstract())) matches public synchronized java.io.InputStream sun.net.www.protocol.http.HttpURLConnection.getInputStream() throws java.io.IOException
[tomcat] 2020-11-29 13:36:58,934 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation DisconnectInstrumentation: ((name(equals(disconnect)) and hasParameter(ofSize(0))) and not(isAbstract())) matches public void sun.net.www.protocol.http.HttpURLConnection.disconnect()
[tomcat] 2020-11-29 13:36:58,953 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker - Init JMX metric tracking with server com.sun.jmx.mbeanserver.JmxMBeanServer@4de8b406
[tomcat] 2020-11-29 13:36:58,953 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation CreateSpanInstrumentation: ((hasSuperType(erasure(is(class java.net.HttpURLConnection))) and not(name(equals(sun.net.www.protocol.https.HttpsURLConnectionImpl)))) and not(isInterface())) matches class javax.net.ssl.HttpsURLConnection
[tomcat] 2020-11-29 13:36:58,954 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation DisconnectInstrumentation: ((hasSuperType(erasure(is(class java.net.HttpURLConnection))) and not(name(equals(sun.net.www.protocol.https.HttpsURLConnectionImpl)))) and not(isInterface())) matches class javax.net.ssl.HttpsURLConnection
[tomcat] 2020-11-29 13:36:58,959 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker - Found mbeans for object name java.lang:type=Memory
[tomcat] 2020-11-29 13:36:58,965 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker - Found composite number attribute HeapMemoryUsage.committed=javax.management.openmbean.CompositeDataSupport(compositeType=javax.management.openmbean.CompositeType(name=java.lang.management.MemoryUsage,items=((itemName=committed,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=init,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=max,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=used,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)))),contents={committed=246415360, init=262144000, max=3706191872, used=100547008})
[tomcat] 2020-11-29 13:36:58,967 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker - Found composite number attribute HeapMemoryUsage.init=javax.management.openmbean.CompositeDataSupport(compositeType=javax.management.openmbean.CompositeType(name=java.lang.management.MemoryUsage,items=((itemName=committed,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=init,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=max,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=used,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)))),contents={committed=246415360, init=262144000, max=3706191872, used=100547008})
[tomcat] 2020-11-29 13:36:58,968 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker - Found composite number attribute HeapMemoryUsage.max=javax.management.openmbean.CompositeDataSupport(compositeType=javax.management.openmbean.CompositeType(name=java.lang.management.MemoryUsage,items=((itemName=committed,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=init,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=max,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=used,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)))),contents={committed=246415360, init=262144000, max=3706191872, used=100547008})
[tomcat] 2020-11-29 13:36:58,969 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker - Found composite number attribute HeapMemoryUsage.used=javax.management.openmbean.CompositeDataSupport(compositeType=javax.management.openmbean.CompositeType(name=java.lang.management.MemoryUsage,items=((itemName=committed,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=init,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=max,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)),(itemName=used,itemType=javax.management.openmbean.SimpleType(name=java.lang.Long)))),contents={committed=246415360, init=262144000, max=3706191872, used=100547008})
[tomcat] 2020-11-29 13:36:58,970 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker$JmxMetricRegistration - Registering JMX metric java.lang:type=Memory HeapMemoryUsage.committed as metric_name: jvm.jmx.test_heap_metric.committed labels: type=Memory
[tomcat] 2020-11-29 13:36:58,972 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker$JmxMetricRegistration - Registering JMX metric java.lang:type=Memory HeapMemoryUsage.init as metric_name: jvm.jmx.test_heap_metric.init labels: type=Memory
[tomcat] 2020-11-29 13:36:58,972 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.report.ApmServerHealthChecker - Starting healthcheck to http://apm-server:1080/
[tomcat] 2020-11-29 13:36:58,972 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker$JmxMetricRegistration - Registering JMX metric java.lang:type=Memory HeapMemoryUsage.max as metric_name: jvm.jmx.test_heap_metric.max labels: type=Memory
[tomcat] 2020-11-29 13:36:58,972 [elastic-apm-remote-config-poller] DEBUG co.elastic.apm.agent.configuration.ApmServerConfigurationSource - Reloading configuration from APM Server http://apm-server:1080/config/v1/agents
[tomcat] 2020-11-29 13:36:58,973 [Attach Listener] DEBUG co.elastic.apm.agent.jmx.JmxMetricTracker$JmxMetricRegistration - Registering JMX metric java.lang:type=Memory HeapMemoryUsage.used as metric_name: jvm.jmx.test_heap_metric.used labels: type=Memory
[tomcat] 2020-11-29 13:36:58,975 [Attach Listener] DEBUG co.elastic.apm.agent.util.ExecutorUtils - A new thread named `elastic-apm-sampling-profiler` was created. The original context class loader of this thread (sun.misc.Launcher$AppClassLoader@70dea4e) has been overridden
[tomcat] 2020-11-29 13:36:58,979 [Attach Listener] INFO  co.elastic.apm.agent.impl.ElasticApmTracer - Tracer switched to RUNNING state
[tomcat] 2020-11-29 13:36:58,990 [elastic-apm-server-healthcheck] DEBUG co.elastic.apm.agent.bci.IndyPluginClassLoaderFactory - Creating plugin class loader for null containing [co.elastic.apm.agent.urlconnection.package-info, co.elastic.apm.agent.urlconnection.SSLContextInstrumentation, co.elastic.apm.agent.urlconnection.HttpUrlConnectionInstrumentation$CreateSpanInstrumentation, co.elastic.apm.agent.urlconnection.UrlConnectionPropertyAccessor, co.elastic.apm.agent.urlconnection.HttpUrlConnectionInstrumentation$DisconnectInstrumentation, co.elastic.apm.agent.urlconnection.HttpUrlConnectionInstrumentation, co.elastic.apm.agent.bci.classloading.LookupExposer]
[tomcat] 2020-11-29 13:36:59,045 [elastic-apm-server-healthcheck] INFO  co.elastic.apm.agent.report.ApmServerHealthChecker - Elastic APM server is available:
[tomcat] 2020-11-29 13:36:59,047 [elastic-apm-server-healthcheck] WARN  co.elastic.apm.agent.report.ApmServerHealthChecker - Failed to parse version of APM server http://apm-server:1080/: Unexpected end of JSON input
[tomcat] 2020-11-29 13:36:59,071 [elastic-apm-remote-config-poller] DEBUG co.elastic.apm.agent.configuration.ApmServerConfigurationSource - This APM Server does not support central configuration. Update to APM Server 7.3+
[tomcat] 2020-11-29 13:36:59,072 [elastic-apm-remote-config-poller] DEBUG co.elastic.apm.agent.configuration.ApmServerConfigurationSource - Scheduling next remote configuration reload in 300s
[tomcat] 2020-11-29 13:36:59,901 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving JSON_WRITER event (sequence 0)
[tomcat] 2020-11-29 13:36:59,901 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Starting new request to http://apm-server:1080/intake/v2/events
[tomcat] 2020-11-29 13:36:59,910 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation TimerTaskInstrumentation: (((false or name(startsWith(co.elastic))) and hasSuperClass(erasure(name(equals(java.util.TimerTask))))) and not(isInterface())) matches class co.elastic.apm.agent.report.IntakeV2ReportingEventHandler$FlushOnTimeoutTimerTask
[tomcat] 2020-11-29 13:36:59,916 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation TimerTaskInstrumentation: (name(equals(run)) and not(isAbstract())) matches public void co.elastic.apm.agent.report.IntakeV2ReportingEventHandler$FlushOnTimeoutTimerTask.run()
[tomcat] 2020-11-29 13:36:59,919 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Scheduling request timeout in 10s
[tomcat] 2020-11-29 13:36:59,920 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving FLUSH event (sequence 1)
[tomcat] 2020-11-29 13:36:59,920 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Flushing 2582 uncompressed 870 compressed bytes
2020-11-29 13:36:56.872  INFO Attaching the Elastic APM agent to 1 org.apache.catalina.startup.Bootstrap -Djava.util.logging.config.file=/usr/local/tomcat/conf/logging.properties -Djava.util.logging.manager=org.apache.juli.ClassLoaderLogManager -Djdk.tls.ephemeralDHKeySize=2048 -Djava.protocol.handler.pkgs=org.apache.catalina.webresources -Dorg.apache.catalina.security.SecurityListener.UMASK=0027 -Dignore.endorsed.dirs= -Dcatalina.base=/usr/local/tomcat -Dcatalina.home=/usr/local/tomcat -Djava.io.tmpdir=/usr/local/tomcat/temp with arguments {}

2020-11-29 13:36:58.980  INFO Done


[tomcat] 2020-11-29 13:37:00,087 [http-nio-8080-exec-2] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation BootstrapDelegationClassLoaderInstrumentation: ((((((not(name(startsWith(java.))) and not(name(startsWith(jdk.)))) and not(name(startsWith(com.sun.)))) and not(name(startsWith(sun.)))) and not(name(contains(Bootstrap)))) and hasSuperType(erasure(is(class java.lang.ClassLoader)))) and not(isInterface())) matches class org.apache.jasper.servlet.JasperLoader
[tomcat] 2020-11-29 13:37:00,092 [http-nio-8080-exec-2] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation BootstrapDelegationClassLoaderInstrumentation: (((name(equals(loadClass)) and returns(erasure(is(class java.lang.Class)))) and (hasParameter(hasTypes(erasures(containing(is(class java.lang.String))))) or hasParameter(hasTypes(erasures(containing(is(class java.lang.String), is(boolean))))))) and not(isAbstract())) matches public java.lang.Class org.apache.jasper.servlet.JasperLoader.loadClass(java.lang.String) throws java.lang.ClassNotFoundException
[tomcat] 2020-11-29 13:37:00,093 [http-nio-8080-exec-2] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation BootstrapDelegationClassLoaderInstrumentation: (((name(equals(loadClass)) and returns(erasure(is(class java.lang.Class)))) and (hasParameter(hasTypes(erasures(containing(is(class java.lang.String))))) or hasParameter(hasTypes(erasures(containing(is(class java.lang.String), is(boolean))))))) and not(isAbstract())) matches public synchronized java.lang.Class org.apache.jasper.servlet.JasperLoader.loadClass(java.lang.String,boolean) throws java.lang.ClassNotFoundException
[tomcat] 2020-11-29 13:37:00,900 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving JSON_WRITER event (sequence 2)
[tomcat] 2020-11-29 13:37:00,900 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Starting new request to http://apm-server:1080/intake/v2/events
[tomcat] 2020-11-29 13:37:00,905 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Scheduling request timeout in 10s
[tomcat] 2020-11-29 13:37:00,905 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving FLUSH event (sequence 3)
[tomcat] 2020-11-29 13:37:00,905 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Flushing 2574 uncompressed 872 compressed bytes
[tomcat] 2020-11-29 13:37:01,905 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving JSON_WRITER event (sequence 4)
[tomcat] 2020-11-29 13:37:01,905 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Starting new request to http://apm-server:1080/intake/v2/events
[tomcat] 2020-11-29 13:37:01,907 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Scheduling request timeout in 10s
[tomcat] 2020-11-29 13:37:01,907 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving FLUSH event (sequence 5)
[tomcat] 2020-11-29 13:37:01,908 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Flushing 2576 uncompressed 871 compressed bytes
2020-11-29 19:37:02,133 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - --> GET http://localhost:32850/simple-webapp/index.jsp
2020-11-29 19:37:02,134 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - X-Forwarded-For: 123.123.123.123, 456.456.456.456
2020-11-29 19:37:02,134 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - --> END GET
[tomcat] 2020-11-29 13:37:02,236 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ConnectionInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Connection))))) and not(isInterface())) matches class org.h2.jdbc.JdbcConnection
[tomcat] 2020-11-29 13:37:02,258 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.PreparedStatement org.h2.jdbc.JdbcConnection.prepareStatement(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,262 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.PreparedStatement org.h2.jdbc.JdbcConnection.prepareStatement(java.lang.String,int,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,264 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.CallableStatement org.h2.jdbc.JdbcConnection.prepareCall(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,265 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.CallableStatement org.h2.jdbc.JdbcConnection.prepareCall(java.lang.String,int,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,266 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.CallableStatement org.h2.jdbc.JdbcConnection.prepareCall(java.lang.String,int,int,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,268 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.PreparedStatement org.h2.jdbc.JdbcConnection.prepareStatement(java.lang.String,int,int,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,269 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.PreparedStatement org.h2.jdbc.JdbcConnection.prepareStatement(java.lang.String,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,270 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.PreparedStatement org.h2.jdbc.JdbcConnection.prepareStatement(java.lang.String,int[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,271 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ConnectionInstrumentation: ((((name(startsWith(prepare)) and returns(erasure(hasSuperType(erasure(name(equals(java.sql.PreparedStatement))))))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.PreparedStatement org.h2.jdbc.JdbcConnection.prepareStatement(java.lang.String,java.lang.String[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,401 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteWithQueryInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcStatement
[tomcat] 2020-11-29 13:37:02,401 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcStatement
[tomcat] 2020-11-29 13:37:02,401 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteUpdateNoQueryInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcStatement
[tomcat] 2020-11-29 13:37:02,402 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation AddBatchInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcStatement
[tomcat] 2020-11-29 13:37:02,402 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteBatchInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcStatement
[tomcat] 2020-11-29 13:37:02,402 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutePreparedStatementInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcStatement
[tomcat] 2020-11-29 13:37:02,431 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.ResultSet org.h2.jdbc.JdbcStatement.executeQuery(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,433 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcStatement.executeUpdate(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,434 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcStatement.execute(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,437 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation AddBatchInstrumentation: (((name(startsWith(addBatch)) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public void org.h2.jdbc.JdbcStatement.addBatch(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,438 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteBatchInstrumentation: ((((name(equals(executeBatch)) or name(equals(executeLargeBatch))) and hasParameter(ofSize(0))) and isPublic()) and not(isAbstract())) matches public int[] org.h2.jdbc.JdbcStatement.executeBatch() throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,440 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcStatement.executeUpdate(java.lang.String,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,441 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcStatement.executeUpdate(java.lang.String,int[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,442 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcStatement.executeUpdate(java.lang.String,java.lang.String[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,443 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcStatement.execute(java.lang.String,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,444 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcStatement.execute(java.lang.String,int[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,446 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcStatement.execute(java.lang.String,java.lang.String[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,451 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteWithQueryInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcPreparedStatement
[tomcat] 2020-11-29 13:37:02,451 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcPreparedStatement
[tomcat] 2020-11-29 13:37:02,451 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteUpdateNoQueryInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcPreparedStatement
[tomcat] 2020-11-29 13:37:02,452 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation AddBatchInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcPreparedStatement
[tomcat] 2020-11-29 13:37:02,452 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecuteBatchInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcPreparedStatement
[tomcat] 2020-11-29 13:37:02,452 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Type match for instrumentation ExecutePreparedStatementInstrumentation: ((not(isInterface()) and hasSuperType(erasure(name(equals(java.sql.Statement))))) and not(isInterface())) matches class org.h2.jdbc.JdbcPreparedStatement
[tomcat] 2020-11-29 13:37:02,484 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutePreparedStatementInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(ofSize(0))) and isPublic()) and not(isAbstract())) matches public java.sql.ResultSet org.h2.jdbc.JdbcPreparedStatement.executeQuery() throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,485 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateNoQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(ofSize(0))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcPreparedStatement.executeUpdate() throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,487 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecutePreparedStatementInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(ofSize(0))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcPreparedStatement.execute() throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,488 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public java.sql.ResultSet org.h2.jdbc.JdbcPreparedStatement.executeQuery(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,490 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation AddBatchInstrumentation: (((name(startsWith(addBatch)) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public void org.h2.jdbc.JdbcPreparedStatement.addBatch(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,491 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcPreparedStatement.executeUpdate(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,492 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcPreparedStatement.execute(java.lang.String) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,499 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteBatchInstrumentation: ((((name(equals(executeBatch)) or name(equals(executeLargeBatch))) and hasParameter(ofSize(0))) and isPublic()) and not(isAbstract())) matches public int[] org.h2.jdbc.JdbcPreparedStatement.executeBatch() throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,502 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcPreparedStatement.executeUpdate(java.lang.String,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,515 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcPreparedStatement.executeUpdate(java.lang.String,int[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,517 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteUpdateWithQueryInstrumentation: ((((name(equals(executeUpdate)) or name(equals(executeLargeUpdate))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public int org.h2.jdbc.JdbcPreparedStatement.executeUpdate(java.lang.String,java.lang.String[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,518 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcPreparedStatement.execute(java.lang.String,int) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,520 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcPreparedStatement.execute(java.lang.String,int[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,522 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.ElasticApmAgent - Method match for instrumentation ExecuteWithQueryInstrumentation: ((((name(equals(execute)) or name(equals(executeQuery))) and hasParameter(hasTypes(with(0 matches erasure(is(class java.lang.String)))))) and isPublic()) and not(isAbstract())) matches public boolean org.h2.jdbc.JdbcPreparedStatement.execute(java.lang.String,java.lang.String[]) throws java.sql.SQLException
[tomcat] 2020-11-29 13:37:02,573 [http-nio-8080-exec-4] DEBUG co.elastic.apm.agent.bci.IndyPluginClassLoaderFactory - Creating plugin class loader for ParallelWebappClassLoader
[tomcat] context: simple-webapp
[tomcat] delegate: false
[tomcat] ----------> Parent Classloader:
[tomcat] java.net.URLClassLoader@1d56ce6a
[tomcat] containing [co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecuteBatchInstrumentation, co.elastic.apm.agent.jdbc.signature.package-info, co.elastic.apm.agent.jdbc.signature.JdbcFilter, co.elastic.apm.agent.jdbc.signature.Scanner$Token, co.elastic.apm.agent.jdbc.signature.SignatureParser, co.elastic.apm.agent.jdbc.signature.Scanner, co.elastic.apm.agent.jdbc.signature.SignatureParser$1, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$8, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$9, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$7, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser, co.elastic.apm.agent.jdbc.helper.package-info, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$10, co.elastic.apm.agent.jdbc.helper.JdbcHelper$1, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$2, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$1$TreeNode, co
[tomcat] .elastic.apm.agent.jdbc.helper.ConnectionMetaData$1, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$HostPort, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$5, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$6, co.elastic.apm.agent.jdbc.helper.JdbcHelper, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$1, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$4, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData, co.elastic.apm.agent.jdbc.helper.ConnectionMetaData$ConnectionUrlParser$3, co.elastic.apm.agent.jdbc.JdbcInstrumentation, co.elastic.apm.agent.jdbc.package-info, co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecuteUpdateNoQueryInstrumentation, co.elastic.apm.agent.jdbc.StatementInstrumentation, co.elastic.apm.agent.jdbc.StatementInstrumentation$AddBatchInstrumentation, co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecutePreparedStatementInstrumentation, co.elastic.apm.agent.
[tomcat] jdbc.StatementInstrumentation$ExecuteWithQueryInstrumentation, co.elastic.apm.agent.jdbc.ConnectionInstrumentation, co.elastic.apm.agent.jdbc.StatementInstrumentation$ExecuteUpdateWithQueryInstrumentation, co.elastic.apm.agent.bci.classloading.LookupExposer]
2020-11-29 19:37:02,670 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - <-- 200 http://localhost:32850/simple-webapp/index.jsp (536ms)
2020-11-29 19:37:02,670 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Set-Cookie: JSESSIONID=ACE6ED0733FE51810FAE4AE8C5DD6232; Path=/simple-webapp; HttpOnly
2020-11-29 19:37:02,670 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Content-Type: text/html;charset=UTF-8
2020-11-29 19:37:02,670 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Content-Length: 47
2020-11-29 19:37:02,670 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Date: Sun, 29 Nov 2020 13:37:02 GMT
2020-11-29 19:37:02,670 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Keep-Alive: timeout=20
2020-11-29 19:37:02,670 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Connection: keep-alive
2020-11-29 19:37:02,671 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - 
2020-11-29 19:37:02,672 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - Hello World!



<html>
<body>

</body>
</html>

2020-11-29 19:37:02,672 [main] INFO  co.elastic.apm.servlet.AbstractServletContainerIntegrationTest - <-- END HTTP (47-byte body)
[tomcat] 2020-11-29 13:37:02,898 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving JSON_WRITER event (sequence 6)
[tomcat] 2020-11-29 13:37:02,898 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Starting new request to http://apm-server:1080/intake/v2/events
[tomcat] 2020-11-29 13:37:02,900 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Scheduling request timeout in 10s
[tomcat] 2020-11-29 13:37:02,900 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Receiving FLUSH event (sequence 7)
[tomcat] 2020-11-29 13:37:02,900 [elastic-apm-server-reporter] DEBUG co.elastic.apm.agent.report.IntakeV2ReportingEventHandler - Flushing 2579 uncompressed 883 compressed bytes
[tomcat] 29-Nov-2020 13:37:03.367 INFO [Thread-3] org.apache.coyote.AbstractProtocol.pause Pausing ProtocolHandler ["http-nio-8080"]
[tomcat] 29-Nov-2020 13:37:03.373 INFO [Thread-3] org.apache.catalina.core.StandardService.stopInternal Stopping service [Catalina]
[tomcat] 29-Nov-2020 13:37:03.382 WARNING [Thread-3] org.apache.catalina.loader.WebappClassLoaderBase.clearReferencesJdbc The web application [simple-webapp] registered the JDBC driver [org.h2.Driver] but failed to unregister it when the web application was stopped. To prevent a memory leak, the JDBC Driver has been forcibly unregistered.
[tomcat] 29-Nov-2020 13:37:03.393 INFO [Thread-3] org.apache.coyote.AbstractProtocol.stop Stopping ProtocolHandler ["http-nio-8080"]
[tomcat] 29-Nov-2020 13:37:03.420 INFO [Thread-3] org.apache.coyote.AbstractProtocol.destroy Destroying ProtocolHandler ["http-nio-8080"]

java.lang.AssertionError: 
Expected size:<1> but was:<0> in:
<[]>

	at co.elastic.apm.servlet.AbstractServletContainerIntegrationTest.getAllReported(AbstractServletContainerIntegrationTest.java:378)
	at co.elastic.apm.servlet.AbstractServletContainerIntegrationTest.assertTransactionReported(AbstractServletContainerIntegrationTest.java:309)
	at co.elastic.apm.servlet.tests.ServletApiTestApp.testTransactionReporting(ServletApiTestApp.java:158)
	at co.elastic.apm.servlet.tests.ServletApiTestApp.test(ServletApiTestApp.java:53)
	at co.elastic.apm.servlet.AbstractServletContainerIntegrationTest.testAllScenarios(AbstractServletContainerIntegrationTest.java:300)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:220)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:53)

```

Reply:
Hmm, that's interesting. I agree that this output does not contain method matches to the `ServletInstrumentation` or the `FilterChainInstrumentation`. 
I wonder if it's indeed the switch to Jakarta EE, as the package remains `javax.servlet` and the types are maintained as well.

Reply:
in jakarta 9 ee all packages start with `jakarta.*` insted of `javax.*`

Reply:
That's right, thanks!!

Reply:
Added #1543 to address Jakarta EE 9 support